### ISSUE TITLE
Migrate away from MSBuild-based NuGet package restore and Missing 'KeyModifiers'

### ISSUE THREAD

**Issue by aloisdg** [delaneyj/three.net#2]
Hello,

I have trouble to build the project on VS2013.
It seems that you are using the old version of nuget, right?
Can we move to the updated version?

I start the migration and everything works until I get an error with Pencil.Gaming : `KeyModifiers`:

> Error 1   The type or namespace name 'KeyModifiers' could not be found (are you missing a using directive or an assembly reference?)

The Pencil.Gaming.GLW3 should contains it, doesn't it?

**Comment by delaneyj** [comment]
It appears to run for me using VS2013, though I had VS2015 installed at home, I can try on my laptop to double check that it runs in VS2013.

Pencil.Gaming at least at this time doesn't have a NuGet package for its builds.  Can you try rebuilding the Pencil.Gaming from source and copying the library reference into Three.Net?

I was using the NuGet installed with 2013.

**Comment by aloisdg** [comment]
[Pencil.Gaming.GLFW3](https://www.nuget.org/packages/Pencil.Gaming.GLFW3/) is not valid, isn't it ? he was uploaded after the last commit on the offical repo. I guessed it was updated and reliable.

**Comment by delaneyj** [comment]
I don't know... I just tried in VS2013 on a new machine and I'm getting all the same errors as you, so going to look into it all right now.

**Comment by aloisdg** [comment]
For the migration, I used a [PowerShell script](https://github.com/owen2/AutomaticPackageRestoreMigrationScript) by @owen2.

**Comment by delaneyj** [comment]
I'm not at my fresh machine to test, the missing .dll/Nuget setup was not fully add (.gitignore stuff) so if you get to it before the morning (Pacific Time, USA) let me know if it works for you.

Btw for clarity for coming to the project the GLW3 build on NuGet is for 32 bit, so I built from source and hard reference.  Really should make a pull request to the Pencil.Gaming repo or relook into OpenTK and see if my original concerns are still valid (mostly multi-windowing issues).

**Comment by aloisdg** [comment]
I just finished the migration. It works well here. Do you think we should open an issue on Pencil.Gaming to ask for an official nuget package? We could make it by ourself but it will be better if an official one is maintained.

**Comment by delaneyj** [comment]
Yes that'd be great, it doesn't seem like its being actively worked on.  Either we need to create a Issue for him to update NuGet, or possible try to get OpenTK back and working.  I stopped using the OpenTK version cause I wanted to support multi-window setups and it just wasn't built with that in mind.  In a perfect world it'd come down to what can both support multi-window and easy cross platform integration.  [Actually I found the original issue](http://www.opentk.com/node/3828) and maybe it has been fixed in the last few months.  Worth investigating either way.
